features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/extensions/publisher/assets/api/apis/api-subscriptions.jag

```diff
@@ -0,0 +1,69 @@
+<%
+/*
+ * Copyright (c) 2015 WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This code is refactored in https://github.com/wso2/carbon-apimgt/commit/d203f97712d5ad50081d93d1efb035bf99ac168c